test/uk/gov/hmrc/nisp/models/SchemeMembershipSpec.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.nisp.models
+
+import org.joda.time.LocalDate
+import uk.gov.hmrc.play.test.UnitSpec
+
+class SchemeMembershipSpec extends UnitSpec {
+
+  "SchemeMembership formatting" should {
+    "return April 2016 for SchemeEndDate (2016,4,5) " in {
+      SchemeMembership(new LocalDate(2015, 4, 6), new LocalDate(2016, 4, 5)).endDateFormatted contains "April 2016"
```

Comment:
The title of these tests doesn't match the tests inside, do they need split into separate tests?
Each test should test the start and end dates maybe?